src/Scripting/CSharpTest/ObjectFormatterTests.cs

```diff
@@ -67,6 +67,13 @@ public void Objects()
             Assert.Equal(@"Sort { aB=-1, ab=1, Ac=-1, Ad=1, ad=-1, aE=1, aF=-1, AG=1 }", str);
         }
 
+        [Fact]
+        public void TupleType()
```

Comment:
Could you also add a test using `Tuple<>` type (as opposed to `ValueTuple`, which is what you have now)?